corehq/pillows/mappings/user_mapping.py

```diff
@@ -232,12 +235,10 @@
                                                                 'type': 'string'}},
                                   'type': 'multi_field'}}}
 
-USER_ES_ALIAS = prefix_for_tests('hqusers')
-
 USER_INDEX_INFO = ElasticsearchIndexInfo(
```

Comment:
What's the end goal for these? Do you plan to remove them completely? I'm just finding it the crossover between the adapters and these classes a bit weird. I almost think that the adapters should either replace these entirely or just reference them.

Reply:
Yes, but this PR doesn't include the replacement, so they need to persist until that's implemented.